.travis.yml

```diff
@@ -4,6 +4,10 @@ python:
   - "3.6"
   - "3.5"
 
+env:
+  - PYSTAN_VERSION=2.17
```

Comment:
If we could test against 2.17.1

Reply:
Would you like me to change?

Reply:
Yes, you can do it after tests are fixed.